File: src/main/java/com/example/TemplateMod.java
package com.example;

import com.example.config.ModConfig;
import net.fabricmc.api.ClientModInitializer;

public class TemplateMod implements ClientModInitializer {

    public static final String MOD_ID = "template_mod";

    @Override
    public void onInitializeClient() {
        ModConfig.initialize();
    }
}

File: src/main/kotlin/com/example/TemplateMod.kt
package com.example

import com.example.config.ModConfig
import net.fabricmc.api.ClientModInitializer

class TemplateMod : ClientModInitializer {

    companion object{
        const val MOD_ID = "template_mod"
    }

    override fun onInitializeClient() {
        ModConfig
    }
}